src/index.html

```diff
@@ -5,8 +5,257 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,700"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/">
+        <img
+          src="images/logo.png"
+          alt="Moyo_logo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="/apple" class="nav__link nav__link--active is-active">apple</a>
```

Comment:
```suggestion
            <a href="#apple" class="nav__link nav__link--active is-active">apple</a>
```
fix everywhere, because now it leads to 404 page (there is no `/apple` page in your app)